packages/core/src/browser/tree/tree-container.ts

```diff
@@ -34,5 +35,8 @@ export function createTreeContainer(parent: interfaces.Container): Container {
 
     child.bind(TreeWidget).toSelf();
     child.bind(TreeProps).toConstantValue(defaultTreeProps);
+
+    child.bind(NoopTreeDecoratorService).toSelf().inSingletonScope();
+    child.bind(TreeDecoratorService).toDynamicValue(ctx => ctx.container.get(NoopTreeDecoratorService)).inSingletonScope();
```

Comment:
`child.bind(TrreDecoratorService).to(NoopTreeDecoratorService).inSingletonScope()` would be sufficient